### PR TITLE
Revise Git Repositories Page in Step 5 (fixes #1524)

### LINKED ISSUE #1524: Revise Git Repositories Page in Step 5
Under Start here section, underscores inside "Clone_with_HTTPS_or_Use_SSH?" are unnecessary and may cause confusion.  "Clone with SSH" is more precise than "Use SSH", and "Clone with SSH" is also shown in the popup when clicking Clone or download button.

"Clone with HTTPS or Use SSH?" shows up under the section Clone Your GitHub Repository username.github.io as well.

### PR DESCRIPTION
**Pull request by YangLou1**
Please check the rawgit link:
https://rawgit.com/YangLou1/YangLou1.github.io/ReviseStep5/#!./pages/vi/vi-github-and-repositories.md

### CODE REVIEW

**Comment by ketruong** [comment]
I think you are not branching for the master branch so you have other edited files in this branch.

**Comment by YangLou1** [comment]
@ketruong Thanks for reminding me. I have fixed it and the PR have only one file changed now.

File: pages/vi/vi-github-and-repositories.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ This is just a summary of the steps that you will need to perform. Please, keep
 
 
 * [Clone Your GitHub Repository username.github.io](#Clone_Your_GitHub_Repository_username.github.io)
-* [Clone with HTTPS or Use SSH?](#Clone_with_HTTPS_or_Use_SSH?)
+* [Clone with HTTPS or Clone with SSH?](#Clone_with_HTTPS_or_Use_SSH?)
```
Comment (xyb994) [inline pages/vi/vi-github-and-repositories.md]: We probably want `#Clone_with_HTTPS_or_Use_SSH?` updated as well.

Comment (YangLou1) [reply pages/vi/vi-github-and-repositories.md]: @xyb994 I have updated it. Please take a look.

**Pull request merged**